src/vmm/src/guest_config/x86_64/static_cpu_templates/mod.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+use versionize::{VersionMap, Versionize, VersionizeError, VersionizeResult};
+use versionize_derive::Versionize;
+
+/// Module with C3 CPU template for x86_64
+pub mod c3;
+/// Module with T2 CPU template for x86_64
+pub mod t2;
+/// Module with T2A CPU template for x86_64
+pub mod t2a;
+/// Module with T2CL CPU template for x86_64
+pub mod t2cl;
+/// Module with T2S CPU template for x86_64
+pub mod t2s;
+
+/// Template types available for configuring the x86 CPU features that map
+/// to EC2 instances.
+#[derive(Debug, Default, Clone, Copy, PartialEq, Eq, Serialize, Deserialize, Versionize)]
+pub enum X86_64StaticCpuTemplate {
+    /// C3 Template.
+    C3,
+    /// T2 Template.
+    T2,
+    /// T2S Template.
+    T2S,
+    /// No CPU template is used.
+    #[default]
+    None,
```

Comment:
Agree, but we can not remove this, because it will break current snapshots. Need to wait for firecracker 2.0 where we would be able to remove static templates all together.